flow/connectors/postgres/client.go

```diff
@@ -428,3 +425,31 @@ func (c *PostgresConnector) generateUpdateStatement(allCols []string, unchangedT
 	}
 	return strings.Join(updateStmts, "\n")
 }
+func (c *PostgresConnector) getTableCounts(tables []string) (int64, error) {
```

Comment:
@heavycrystal  replace this with count estimates: https://wiki.postgresql.org/wiki/Count_estimate, counts can be really slow for some of the tables we are mirroring.